src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/Commands.ApplicationInsights.csproj

```diff
@@ -0,0 +1,221 @@
+    <Compile Include="ApplicationInsights\SetApplicationInsightsDailyCap.cs" />
+    <Compile Include="ApplicationInsights\SetApplicationInsightsPricingPlan.cs" />
+    <Compile Include="ApplicationInsights\SetApplicationInsightsContinuousExport.cs" />
+    <Compile Include="ApplicationInsights\NewApplicationInsights.cs" />
+    <Compile Include="Helper\Utilities.cs" />
+    <Compile Include="Models\PSApplicationInsightsComponent.cs" />
+    <Compile Include="Models\PSApiKey.cs" />
+    <Compile Include="Models\PSDailyCapStatus.cs" />
+    <Compile Include="Models\PSDailyCap.cs" />
+    <Compile Include="Models\PSPrcingPlan.cs" />
+    <Compile Include="Models\PSExportConfiguration.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="ApplicationInsights\GetApplicationInsights.cs" />
+    <Compile Include="ApplicationInsights\RemoveApplicationInsights.cs" />
+    <Compile Include="ApplicationInsights\ApplicationInsightsBaseCmdlet.cs" />
+    <Compile Include="ApplicationInsights\ApplicationInsightsManagementClient.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\AzureRM.ApplicationInsights.psd1">
+      <Link>AzureRM.ApplicationInsights.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.ApplicationInsights.dll-Help.psd1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.ApplicationInsights.format.ps1xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
+      <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
+      <Name>Commands.Common.Authentication.Abstractions</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Storage\Commands.Common.Storage.csproj">
+      <Project>{65c3a86a-716d-4e7d-ab67-1db00b3bf72d}</Project>
+      <Name>Commands.Common.Storage</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.Resources\Commands.Resources.csproj">
```

Comment:
@frankguodongchen you should not be referencing the Resources or Tags projects. You can find common code for these projects in [`Commands.ResourceManager.Common`](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Common/Commands.ResourceManager.Common) (Resources is in the `Generated` folder and Tags is in the `Tags` folder)

Reply:
removed. thanks!